src/openloco/map/tilemgr.cpp

```diff
@@ -200,12 +201,10 @@ namespace openloco::map::tilemgr
     }
 
     // 0x004CBE5F
-    void map_invalidate_tile_full(uint16_t x, uint16_t y)
+    void map_invalidate_tile_full(coord_t x, coord_t y)
```

Comment:
You might as well change this type to `map::map_pos` that will ensure that the callers and compiler know it is a map coordinate space that should be passed to the function and prevents unnecessary constructions.

Reply:
Yes, I will do it.